# Msg that stores exogenous actions info

string action_name
string agent_object
bool status
float32 value